### PR TITLE
Add Cypress tests for vaos service request submissions

### PR DESCRIPTION
**Pull request by jbalboni**
## Description
This PR
- Adds two new Cypress tests for the request flow with the vaos service requests flag on

## Testing done
Cypress testing

## Acceptance criteria
- [ ] Build passes

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vaos/tests/e2e/community-care.cypress.spec.js [hunk 1]
```diff
@@ -445,3 +452,259 @@ describe('VAOS community care flow', () => {
     cy.get('va-alert').contains('We’re reviewing your request');
   });
 });
+
+describe('VAOS community care flow using VAOS service', () => {
```
Comment (jbalboni) [inline src/applications/vaos/tests/e2e/community-care.cypress.spec.js:456]: I wrote these slightly differently than our other Cypress tests.

Normally, we have all the mocked data set up in a bunch of nested helper functions. These are hard to follow and so instead I've minimized the helper function code in favor of putting a lot of the mocking in the test code. The tests are longer, but hopefully they're easier to adjust in the future.

I'm happy to talk about this and see if we prefer this or the old style.

**Review by LeKeve — APPROVED** [review]
